.github/workflows/wheel_macos_x86_64.yml

```diff
@@ -156,16 +157,15 @@ jobs:
           CIBW_BEFORE_ALL_MACOS: |
             brew uninstall --force oclint
             brew install libomp
-            brew install llvm
 
           # Python build settings
           CIBW_BEFORE_BUILD: |
-            pip install pybind11 ninja cmake~=3.24.0
+            python -m pip install pybind11 ninja cmake~=3.24.0 setuptools
```

Comment:
I'm aware that the error log was pointing to some problem with setuptools.
But do we still need to install it after changing the compiler and os?

Reply:
I believe it is worth including. It should be autoinstalled, but this seemed to not be the case. I can try removing it and validating again.